manage_user.php

```diff
@@ -1,92 +1,124 @@
+            <input type="text" name="email" id="email" class="form-control" value="<?php echo isset($meta['email']) ? $meta['email'] : '' ?>" required autocomplete="off">
+        </div>
+        <div class="form-group">
+            <label for="password">Password</label>
+            <input type="password" name="password" id="password" class="form-control" value="" autocomplete="off">
+            <small><i>Leave this blank if you don't want to change the password.</i></small>
+        </div>
+        <div class="form-group">
+            <label for="" class="control-label">Avatar</label>
+            <div class="custom-file">
+                <input type="file" class="custom-file-input rounded-circle" id="customFile" name="img" onchange="displayImg(this,$(this))">
+                <label class="custom-file-label" for="customFile">Choose file</label>
             </div>
-		</div>
-		<div class="form-group d-flex justify-content-center">
-			<img src="<?php echo isset($meta['avatar']) ? 'assets/uploads/'.$meta['avatar'] :'' ?>" alt="" id="cimg" class="img-fluid img-thumbnail">
-		</div>
-		
-
-	</form>
+        </div>
+        <div class="form-group d-flex justify-content-center">
+            <img src="<?php echo isset($meta['avatar']) ? 'assets/uploads/'.$meta['avatar'] :'' ?>" alt="" id="cimg" class="img-fluid img-thumbnail">
+        </div>
+        <button type="submit" class="btn btn-primary">Save</button>
+    </form>
 </div>
 <style>
-	img#cimg{
-		height: 15vh;
-		width: 15vh;
-		object-fit: cover;
-		border-radius: 100% 100%;
-	}
+    img#cimg{
+        height: 15vh;
+        width: 15vh;
+        object-fit: cover;
+        border-radius: 100% 100%;
+    }
 </style>
 <script>
-	function displayImg(input,_this) {
-	    if (input.files && input.files[0]) {
-	        var reader = new FileReader();
-	        reader.onload = function (e) {
-	        	$('#cimg').attr('src', e.target.result);
-	        }
+    function displayImg(input, _this) {
+        if (input.files && input.files[0]) {
+            var reader = new FileReader();
+            reader.onload = function (e) {
+                $('#cimg').attr('src', e.target.result);
+            }
 
-	        reader.readAsDataURL(input.files[0]);
-	    }
-	}
-	$('#manage-user').submit(function(e){
-		e.preventDefault();
-		start_load()
-		$.ajax({
-			url:'ajax.php?action=update_user',
-			data: new FormData($(this)[0]),
-		    cache: false,
-		    contentType: false,
-		    processData: false,
-		    method: 'POST',
-		    type: 'POST',
-			success:function(resp){
-				if(resp ==1){
-					alert_toast("Data successfully saved",'success')
-					setTimeout(function(){
-						location.reload()
-					},1500)
-				}else{
-					$('#msg').html('<div class="alert alert-danger">Username already exist</div>')
-					end_load()
-				}
-			}
-		})
-	})
+            reader.readAsDataURL(input.files[0]);
+        }
+    }
 
-</script>
\ No newline at end of file
+    $('#manage-user').submit(function(e){
```

Comment:
This function calls an empty url - but I see the query is written on top - how are you testing any of your changes?
Going forward I need screenshots added to the PR to see what output you are seeing on screen